data/brands/amenity/bicycle_rental.json

```
@@ -1726,6 +1726,18 @@
         "network": "HuiziToluca"
       }
     },
+        {
+      "displayName": "I'Velo",
+      "id": "",
+      "locationSet": {"include": ["ro"]},
+      "tags": {
+        "amenity": "bicycle_rental",
+        "brand": "I'Velo",
+        "brand:wikidata": "Q119495434",
+        "network": "I'Velo",
+        "operator": "Asociația Green Revolution"
+        "matchNames": "Bucharest bike share"
+      }
```

`matchNames` shouldn't be in the tags, but on the item.
```suggestion
      },
      "matchNames": ["Bucharest bike share"]
      }
```